security/advancedtls/testdata/crl/provider_create.sh

```diff
@@ -72,5 +72,35 @@ openssl ca -gencrl                       \
   -out provider_crl_server_revoked.pem   \
   -config provider_crl.cnf
 
+openssl genrsa                                      \
+  -out provider_malicious_client_trust_key.pem 4096
+
+SKI=$(openssl x509 -in provider_client_trust_cert.pem \
```

Comment:
SKI -> SubjectKeyIdentifier

Reply:
Since it's internal var for bash script, I'd prefer to keep it short.

Reply:
I'd prefer to go with the full name - longer names are free, and this way I don't have to take time to search what `SKI` could stand for if I forget

Reply:
Renamed